mobile/src/components/chat-conversation/components/messages-list/messages-list.tsx

```diff
@@ -23,9 +25,24 @@ const MessagesList: FC<Props> = ({ currentUserId, messages }) => {
     }),
   );
 
+  const scrollToDown = (): void => {
+    chatRef.current?.scrollToLocation({
+      animated: true,
```

Comment:
It's better to remove animation here, because we can have very long history and we don't want user to wait for it